openquake/calculators/tests/event_based_risk_test.py

```diff
@@ -74,21 +75,18 @@ def assert_stats_ok(self, pkg, job_ini, individual_curves='false'):
     def test_case_1(self):
         self.run_calc(case_1.__file__, 'job.ini',
                       exports='csv', individual_curves='false')
-        ekeys = [
-            ('rcurves-stats', 'xml'),
-            ('rcurves-stats', 'geojson'),
-
-            ('loss_maps-stats', 'csv'),
-            ('loss_maps-stats', 'xml'),
-            ('loss_maps-stats', 'geojson'),
-
-            ('agg_curve-stats', 'xml'),
-        ]
+        ekeys = [('agg_curve-stats', 'xml')]
         for ekey in ekeys:
             for fname in export(ekey, self.calc.datastore):
                 self.assertEqualFiles(
                     'expected/%s' % strip_calc_id(fname), fname)
 
+        fnames = export(('loss_maps-stats', 'csv'), self.calc.datastore)
+        if REFERENCE_OS:
```

Comment:
Should we have the sanity check `assert fnames` also here and in all the other occurrences?